modules/conversion-center/classes/providers/social-network-provider.php

```diff
@@ -233,4 +233,27 @@ private static function init_social_networks_array_if_empty(): void {
 			'name' => 'skype',
 		];
 	}
+
+	public static function build_email_link( array $data, string $prefix ) {
+		$email = $data[ $prefix . '_mail' ] ?? '';
+		$subject = $data[ $prefix . '_mail_subject' ] ?? '';
+		$body = $data[ $prefix . '_mail_body' ] ?? '';
+
+		if ( ! $email ) {
+			return '';
+		}
+
+		$link = 'mailto:' . $email;
+
+		if ( $subject ) {
+			$link .= '?subject=' . $subject;
+		}
+
+		if ( $body ) {
+			$link .= $subject ? '&' : '?';
+			$link .= 'body=' . $body;
+		}
+
+		return $link;
+	}
```

Comment:
@mserino There is already a function `build_email_link()` in `elementor/modules/conversion-center/classes/render/render-base.php`

Can you add it to the trait and use it in both these places?

Reply:
Oh sorry, I see you deleted it in the other file!